test/lib/Edamam_Api_Wrapper_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
```

Comment:
Good work on testing the positive and negative cases using the API.

I might also add a test for what happens with an API request with no results.  Like searching for "asdf"